src/plugins/foxPage/FoxCommon.tsx

```diff
@@ -0,0 +1,12 @@
+export type OtherOpportunity = {
```

Comment:
Also not sure about the OtherOpportunity terminology, it makes sense for the UI since it's user-facing, but for us, it's just an opportunity.

Reply:
But we called the main opportunity `FoxOpportunity` which is a complete different shape, maybe we could merge it somehow?

Reply:
IMO it makes sense that they are separate, but typing-wise, `OtherOpportunity` just isn't very descriptive. Fine to keep the component itself named like so